samples/msgext-unfurling-ac-loop-components/csharp/README.md

```diff
@@ -0,0 +1,191 @@
+---
+page_type: sample
+description: This sample shows messaging extension with adaptive card-based loop components functionality.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "09-06-2023 20:32:25"
+urlFragment: officedev-microsoft-teams-samples-msgext-unfurling-ac-loop-components-csharp
+---
+
+# Messaging Extension with Adaptive Card Loop Components
+
+If you copy and paste a link from `https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main` or your ngrok base url, it wil unfurl inside compose area.
+
+## Included Features
+* Message Extensions
+* Search Commands
+* Link Unfurling
+* Adaptive card based loop component
+
+## Interaction with teams
+![msgext-unfurling-ac-loop-components](Images/msgext-unfurling-ac-loop-components.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Msgext-Unfurling-Adaptive-Card-Loop-Components:** [Manifest](/samples/msgext-unfurling-ac-loop-components/csharp/demo-manifest/msgext-unfurling-ac-loop-components.zip)
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution
```

Comment:
Remove en-us from all the URLS but make sure those are not broken

Reply:
Fixed